notebook-images/base/jupyter-minimal-gpu-notebook-imagestream.yaml

```diff
@@ -23,6 +23,8 @@ spec:
       kind: DockerImage
       name: quay.io/opendatahub/workbench-images:cuda-jupyter-minimal-ubi9-python-3.9-2023a-weekly
     name: "2023.1-cuda-11.8"
+    importPolicy:
+      scheduled: true
```

Comment:
yes, that setting for the weekly-changing tags assures that the imagestream status-section status.tags[x].items.dockerImageReference always refers to the latest digest value, as well as status.tags[x].items.image.